pkg/commands/ignore.go

```diff
@@ -263,13 +263,14 @@ $ bearer ignore migrate`,
 			if err != nil {
 				return fmt.Errorf("flag error: %s", err)
 			}
+
 			configFilePath := viper.GetString(flag.ConfigFileFlag.ConfigName)
```

Comment:
Curious here: is the following equivalent? 

```suggestion
			configFilePath := viper.GetString(options.GeneralOptions.ConfigFile)
```

We use `flag.ConfigFileFlag.ConfigName` in the commands/scan.go but perhaps this is for performance or other reasons. Any light to be shed @didroe or @cfabianski? 🙏

Reply:
In this case it's equivalent (without the `viper.GetString`). But in the scan command, we read the options from the config file. So we have to read it's location before building the options.

Reply:
Ahh, thanks for the explanation. So, if we wanted to support reading options from the config file in this case too, we should leave it as the following?

```golang
configFilePath := viper.GetString(flag.ConfigFileFlag.ConfigName)
```

Reply:
Yeah. Basically we'd need to do [this](https://github.com/Bearer/bearer/blob/911c5c161774f639f1d34e7dc25a7b4596b8df82/pkg/commands/scan.go#L79-L88) before calling `<FLAGS>.ToOptions`. And therefore you'd need to grab the filename manually first.